easybuild/easyconfigs/r/reframe-cscs-tests/reframe-cscs-tests-24.03.eb

```diff
@@ -0,0 +1,38 @@
+# @author: manitart
+
+easyblock = 'Tarball'
+
+name = 'reframe-cscs-tests'
+version = '24.03'
+
+homepage = 'https://github.com/eth-cscs/cscs-reframe-tests'
+description = 'ReFrame test suite of CSCS.'
+
+toolchain = SYSTEM
+
+dependencies = [('reframe', '4.2.0')]
```

Comment:
```suggestion
dependencies = [('reframe', '4.5.1')]
```

Reply:
We have `reframe-4.5.1.eb` in the repository, is it possible to use it as a dependency here?